modules/element-manager/assets/js/app-editor-one/components/SearchFilters.js

```diff
@@ -0,0 +1,165 @@
+import {
+	Button,
+	Stack,
+	Box,
+	TextField,
+	Select,
+	MenuItem,
+	FormControl,
+	Divider,
+} from '@elementor/ui';
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+
+export const SearchFilters = ( {
+	searchKeyword,
+	onSearchChange,
+	filterByPlugin,
+	onPluginFilterChange,
+	filterByStatus,
+	onStatusFilterChange,
+	plugins,
+	usageIsLoading,
+	usageData,
+	widgetsDisabledCount,
+	onScanUsage,
+	onDeactivateUnused,
+	onEnableAll,
+	onSaveChanges,
+	isSaving,
+	hasUnsavedChanges,
+} ) => {
+	return (
+		<Stack
+			direction="row"
+			alignItems="center"
+			justifyContent="space-between"
+			sx={ {
+				position: 'sticky',
+				top: '32px',
+				backgroundColor: 'var(--e-one-palette-background-default)',
```

Comment:
```javascript
sx={ { maxWidth: 16, minWidth: 16 } }
```

It seems odd to use maxWidth and minWidth. Possibly check if there is a better way to force a fixed width inside this layout.